framework/doc/content/source/base/MooseObject.md

```diff
@@ -0,0 +1,43 @@
+# MooseObject
```

Comment:
Wow! First time we've implemented a doc for MooseObject?!

Reply:
Yeah, I was a bit surprised myself...